cypress/support/pages/admin_console/manage/events/EventsPage.ts

```diff
@@ -0,0 +1,53 @@
+export default class EventsPage {
+  searchEventDrpDwn: string = ".pf-c-dropdown__toggle";
+  searchEventDrpDwnBtn: string =
+    ".keycloak__user_events_search_selector_dropdown__toggle";
+  searchForm: string = ".pf-c-dropdown__menu";
+  userIdInputFld: string = "userId-searchField";
+  eventTypeDrpDwnFld: string = "event-type-searchField";
+  clientInputFld: string = "client-searchField";
+  dateFromInputFld: string = "dateFrom-searchField";
+  dateToInputFld: string = "dateTo-searchField";
+  searchEventsBtn: string = "search-events-btn";
+  eventTypeList: string = ".pf-c-form-control";
+  eventTypeOption: string = ".pf-c-select__menu-item";
+  eventTypeBtn: string = ".pf-c-button.pf-c-select__toggle-button.pf-m-plain";
+  eventsPageTitle: string = ".pf-c-title";
```

Comment:
There is no need to specify a type for these fields as TypeScript can infer the type from the assigned value (in this case a string).
```suggestion
  searchEventDrpDwn = ".pf-c-dropdown__toggle";
  searchEventDrpDwnBtn =
    ".keycloak__user_events_search_selector_dropdown__toggle";
  searchForm = ".pf-c-dropdown__menu";
  userIdInputFld = "userId-searchField";
  eventTypeDrpDwnFld = "event-type-searchField";
  clientInputFld = "client-searchField";
  dateFromInputFld = "dateFrom-searchField";
  dateToInputFld = "dateTo-searchField";
  searchEventsBtn = "search-events-btn";
  eventTypeList = ".pf-c-form-control";
  eventTypeOption = ".pf-c-select__menu-item";
  eventTypeBtn = ".pf-c-button.pf-c-select__toggle-button.pf-m-plain";
  eventsPageTitle = ".pf-c-title";
```

Reply:
done, thanks